src/main/java/mil/dds/anet/utils/Utils.java

```diff
@@ -207,9 +216,45 @@ public static String sanitizeHtml(String input) {
     if (input == null) {
       return null;
     }
-    return POLICY_DEFINITION.sanitize(input);
+    return HTML_POLICY_DEFINITION.sanitize(input);
   }
 
+  public static String sanitizeJSON(String inputJson) throws JsonProcessingException {
+    String sanitizedJson = JsonSanitizer.sanitize(inputJson);
+    JsonNode jsonTree = mapper.readTree(sanitizedJson);
+    internalSanitizeJSONforHTML(jsonTree);
+    return mapper.writeValueAsString(jsonTree);
+  }
+
+  private static void internalSanitizeJSONforHTML(JsonNode jsonNode) {
+    if (jsonNode.isObject()) {
+      ObjectNode objectNode = (ObjectNode) jsonNode;
+      Iterator<Map.Entry<String, JsonNode>> iter = objectNode.fields();
+      while (iter.hasNext()) {
+        Map.Entry<String, JsonNode> entry = iter.next();
+        if (entry.getValue().isTextual()) {
+          objectNode.put(entry.getKey(), sanitizeHtml(entry.getValue().asText()));
+        } else {
+          internalSanitizeJSONforHTML(entry.getValue());
+        }
+        String sanitizedKey = sanitizeHtml(entry.getKey());
+        if (!entry.getKey().equals(sanitizedKey)) {
+          objectNode.remove(entry.getKey());
+          objectNode.set(sanitizedKey, entry.getValue());
```

Comment:
Does `entry.getValue()` here get the value possibly modified above?

Reply:
good catch

Reply:
Be sure to add a test case for it!